code/controllers/subsystems/assets.dm

```diff
@@ -3,64 +3,37 @@
 /datum/controller/subsystem/assets
 	name = "Assets"
 	init_order = SS_INIT_ASSETS
-	flags = SS_BACKGROUND
+	flags = SS_NO_FIRE
 	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
-	wait = 1
-	var/list/cache = list()
-	var/list/target_clients = list()
-	var/list/currentrun
+	var/list/datum/asset_cache_item/cache = list()
+	var/list/preload = list()
+	var/datum/asset_transport/transport = new()
 
 /datum/controller/subsystem/assets/New()
 	NEW_SS_GLOBAL(SSassets)
+	var/newtransporttype = /datum/asset_transport
```

Comment:
we don't have a hook for when config's loaded in subsystems, so i guess we just use initialize instead

```suggestion

/datum/controller/subsystem/assets/Initialize()
	var/newtransporttype = /datum/asset_transport
```